sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_log_auth_events_v1/metadata.yaml

```diff
@@ -11,9 +11,11 @@ labels:
   application: fxa
   incremental: true
   schedule: daily
-scheduling:
-  dag_name: bqetl_fxa_events
-  referenced_tables: []
+# Query descheduled as a direct result of AWS migration to GCP
+# on 27th September 2023 the last AWS instances were spun down.
+# scheduling:
+#   dag_name: bqetl_fxa_events
```

Comment:
These `fxa_log_*` ETLs included non-Amplitude events, but the new GCP-based ETLs only include Amplitude events.  Do we need new GCP-based equivalents of these `fxa_log_*` ETLs?

Reply:
I'm not aware of these being used anywhere. The only `fxa_log_*` tables I know that are being used are `fxa_log_device_command_events_v1` and `fxa_log_device_command_events_v2`. 

I do not see any usage of `fxa_log_auth_events_v1` and `fxa_content_events_v1`
other than `fxa_log_performance_events` which in turn I do not see where it is being used nor we had anyone reaching out about it being outdated.